proc_IR.py

```diff
@@ -19,25 +19,17 @@
         ('200212_IR_3_30dBm', 'test_equip', 'signal', 
             'spincore_IR_v1'),
         ]:
+    fl.basename = searchstr
     s = find_file(searchstr, exp_type=exp_type,
             expno=nodename,
             postproc=postproc, lookup=postproc_dict,
             dimname='indirect')
     s *= exp(-1j*s.fromaxis('indirect')*clock_correction)
-    #logger.info(strm(s.dimlabels))
-    #{{{rough centers data
-    fl.next('filtered + rough centered data')
+    #{{{filter data
     s = s['t2':(-filter_bandwidth/2,filter_bandwidth/2)]
-    s.ift('t2')
-    rough_center = abs(s).convolve('t2',0.01).mean_all_but(
-            't2').argmax('t2').item()
-    s.setaxis(t2-rough_center)
```

Comment:
on looking at the code more recently, rough centering should really not be required

Reply:
but clearly do check if it works, and add back in if it's needed

Reply:
took out as it works the same without it